core/templates/components/forms/schema-based-editors/schema-based-editor.component.ts

```diff
@@ -89,6 +89,10 @@ implements AfterViewInit, ControlValueAccessor, Validator {
   ngAfterViewInit(): void {
     let angularJsFormController: angular.IFormController = angular.element(
       this.elementRef.nativeElement).controller('form');
+    // This throws "Object is possibly undefined." The type undefined
+    // comes here from NgForm dependency. We need to suppress this
+    // error because of strict type checking.
+    // @ts-ignore
     this.form.statusChanges.subscribe((validationStatus) => {
       if (angularJsFormController === null ||
```

Comment:
Unable to solve this "Object is possibly undefined." as it comes from NgForm. Need help.